caluma/caluma_workflow/serializers.py

```diff
@@ -256,105 +256,61 @@ def update(self, instance, validated_data):
         return instance
 
 
-class CompleteWorkItemSerializer(SendEventSerializerMixin, serializers.ModelSerializer):
+class CompleteWorkItemSerializer(serializers.ModelSerializer):
     id = serializers.GlobalIDField()
 
     def validate(self, data):
-        user = self.context["request"].user
-        validators.WorkItemValidator().validate(
-            status=self.instance.status,
-            child_case=self.instance.child_case,
-            task=self.instance.task,
-            case=self.instance.case,
-            document=self.instance.document,
-            info=self.context["info"],
-        )
-        data["status"] = models.WorkItem.STATUS_COMPLETED
-        data["closed_at"] = timezone.now()
-        data["closed_by_user"] = user.username
-        data["closed_by_group"] = user.group
-        return super().validate(data)
+        try:
```

Comment:
I think the try/except isn't really needed here - or why do you catch a validation error just to raise one again (as far as I know, they're the same type anyway, despite coming from different modules)

Reply:
Not sure about that: https://github.com/encode/django-rest-framework/blob/f0dbf0a264677f2a53faab402ff49f442fc4383a/docs/community/3.0-announcement.md#using-serializersvalidationerror

Reply:
"whenever raising validation errors you should prefer using the `serializers.ValidationError` exception class, and not Django's built-in exception."

Reply:
Fair enough! Keep it as is then